scripts/run_texstudio.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+texstudio ../latex/main.tex
```

Comment:
Are there causes?

Reply:
It's literally a one-liner that opens the main tex file, and a one which can be done graphically (and due to the nature of most LaTeX editors, it will be done graphically). So, the reason is: this script might make people feel that something special happens under the hood so they'll figure they NEED texstudio for the docs to compile, while in reality you should just tell them which file to open.